ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -27,6 +27,10 @@ let prototyped_of_field = function
       Some "23.14.0"
   | "Repository", "gpgkey_path" ->
       Some "22.12.0"
+  | "Certificate", "fingerprint_sha1" ->
+      Some "24.18.0-next"
```

Comment:
this is out of date, should be 24.18.1-next

Reply:
24.19.1-next now